backend/algorithm.js

```diff
@@ -0,0 +1,56 @@
+const symptoms = ['Fever', 'Cough', 'Headache', 'Diarrhea', 'Body Pain'];
+const medications = ['Paracetamol', 'Amoxicillin', 'Ciprofloxacin', 'Ibuprofen', 'Metronidazole', 'Artemether/Lumefantrine', 'Vitamin C', 'Diclofenac'];
+
+const ageGroupWeights = {
+    child: [4, 4, 3, 3, 2],
+    teenager: [3, 3, 4, 2, 2],
+    adult: [3, 3, 4, 2, 4],
+    elder: [2, 4, 2, 3, 5]
+  };
+
+const medicationWeights = [
+    [4, 1, 5, 0, 4], // Paracetamol
```

Comment:
Can this be a dictionary (similar to ageGroupWeights)? That way the name of the medication can be in the code itself

Reply:
On it!

Reply:
I'll change all other arrays  that attribute to medicine to dictionaries, just to ensure consistency.